projects/packages/publicize/src/class-publicize.php

```diff
@@ -139,8 +140,8 @@ class="button-connector"
 	 * @param bool      $force_delete Whether to skip permissions checks.
 	 * @return false|void False on failure. Void on success.
 	 */
-	public function disconnect( $service_name, $connection_id, $_blog_id = false, $_user_id = false, $force_delete = false ) {
-		return Jetpack_Keyring_Service_Helper::disconnect( $service_name, $connection_id, $_blog_id, $_user_id, $force_delete );
+	public function disconnect( $service_name, $connection_id, $_blog_id = false, $_user_id = false, $force_delete = false ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
Which variable is unused here? It looks like they're all passed to `disconnect`

Reply:
Resolved in 028e7a5